### ISSUE TITLE
Enable the ability to 'try anyway' on a page that does not have '.csv' extension.

### ISSUE THREAD

**Issue by chrishalbert** [rubenvar/csv-reader#7]
First off, I want to say thank you for your contributions. This is a great project and makes for a very clean user interface.

**Feature Request:** 
Some websites have a path that look like `some.url/file.csv/view` which the csv reader does not allow. I'd like to suggest allowing a user to try to use the reader anyway for cases like these. I drafted up an option, but feel free to suggest changes, implement as you see fit, or ignore completely :) 

![image](https://user-images.githubusercontent.com/16158060/112167094-95ec2a00-8bc6-11eb-822f-8837306f5133.png)

I unpacked and made edits to accommodate this functionality. When I did this, I was getting errors due to different method signatures with the browser.tabs(). I think I was missing a step. Anyhow, thanks again for your contribution.

**Comment by rubenvar** [comment]
@chrishalbert thank you very much for your suggestion and the work you have already done, I just saw your PR :) I think it's a great idea to have this option.

I'm going to test it during these next few days, and if everything works alright I will add it in the next update of the addon, together with some styling changes I've been working on locally, which I hope will be ready soon.

Thanks again!

**Comment by rubenvar** [comment]
hey @chrishalbert, I already added your idea and it will be published on the next update of the extension, thanks again!

I saw you adapted the `.query()` methods in `browser.tabs` to use callbacks instead of `.then()`, but had to revert it as I could not make it work in Chrome (I was getting an error from the polyfill).

**Comment by chrishalbert** [comment]
Hey @rubenvar - I saw the polyfill library and was unsure if there was a step I needed to do because the library worked just fine until I unpacked and loaded that locally in development mode. 

Is there a step that needs to happen after making edits during development? Maybe that could be added to the README? (I may be interested in contributing to some of your other ideas in the future).

Regardless, thank you so much for adding the feature! I'll be suggesting this to my team as this helps our debugging on a project. 🥂

**Comment by rubenvar** [comment]
I just sent the updated extension for revision. The new styling and options should be available once it's approved.

I updated the README with the dev process I follow:

- For Chrome I just load the unpacked extension and it should work when the polyfill for `Promises` is included in the files (I just realized the polyfill was not being committed before, so I guess that's what was caussing your issues? I'm really sorry about that 🙇‍♂️).
- And for Firefox, I use the [web-ext](https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/) extension by Mozilla and follow their dev and build process.

I hope this helps!

I'm closing this issue now, let me know if something is not clear, and of course any help and suggestions are much appreciated
